src/views/ConfigEditor.tsx

```diff
@@ -14,13 +19,30 @@ const ConfigEditor = (props: ConfigEditorProps) => {
   const { jsonData, secureJsonData, secureJsonFields } = options;
   const secureSettings = (secureJsonData || {}) as GitHubSecureJsonData;
   const styles = useStyles2(getStyles);
+  const WIDTH_LONG = 40;
+  const authOptions = [
+    { label: 'Personal Access Token', value: 'personal-access-token' },
+    { label: 'GitHub App', value: 'github-app' },
+  ];
+  const licenseOptions = [
+    { label: 'Basic', value: 'github-basic' },
+    { label: 'Enterprise', value: 'github-enterprise' },
+  ];
 
-  const [selectedLicense, setSelectedLicense] = useState(jsonData.githubUrl ? 'github-enterprise' : 'github-basic');
   const [isOpen, setIsOpen] = useState(true);
+  const [selectedLicense, setSelectedLicense] = useState(jsonData.githubUrl ? 'github-enterprise' : 'github-basic');
```

Comment:
Maybe we could have these `github-enterprise` and `github-basic` as variables, as we are using them multiple times, to prevent incorrect spelling somewhere. 

Also maybe the same with 'personal-access-token` and `github-app`